Assets/Scripts/Models/Furniture.cs

```diff
@@ -279,6 +351,12 @@ private set
         }
     }
 
+    /// <summary>
+    /// Used to place furniture in a certain position
+    /// </summary>
+    /// <param name="proto">The prototype furniture to place</param>
+    /// <param name="tile">The tile to place the furniture on</param>
```

Comment:
The **base** tile to place the furniture on, **The tile will be the bottom left corner of the furniture** (to check)